stimela/utils/xrun_poll.py

```diff
@@ -61,7 +61,7 @@ def poll(self, timeout=5, verbose=False):
                     if verbose:
                         self.log.debug("poll(): retrying")
                 else:
-                    raise
+                    raise ioerr
```

Comment:
Probably superfluous -- raise by itself just re-raises the current exception. Which in this case is ``ioerr``.